### DISCUSSION TITLE
Is there the possibility that we are able to inject our own addons?

### DISCUSSION THREAD

**Discussion by redstone-dev** [ScratchAddons/ScratchAddons#1046] [Discussion]
From the README:
> Scratch Addons by itself is just an addon loader.

So wait, are we able to inject our own addons? If so, how do we do that?

**Comment by WorldLanguages** [comment]
Not directly. Because of security concerns, addons update together with the extension (the "addon loader") in both extension web stores. So we encourage anyone that created an addon to pull request to this repository.
Here's a guide om [creating an addon](https://github.com/ScratchAddons/ScratchAddons/wiki/Creating-an-addon).

**Reply by ghost** [reply]
> Not directly. Because of security concerns, addons update together with the extension (the "addon loader") in both extension web stores. So we encourage anyone that created an addon to pull request to this repository.
> Here's a guide om [creating an addon](https://github.com/ScratchAddons/ScratchAddons/wiki/Creating-an-addon).

*on*
